pkg/appmanager/as3Manager.go

```diff
@@ -452,6 +471,22 @@ func (as3RestClient *As3RestClient) restCallToBigIP(method string, route string,
 
 }
 
+// Read certificate from configmap
+func (appMgr *Manager) getCertFromConfigMap(cfgmap string) {
+
+	certs := ""
+	cm, err := appMgr.kubeClient.CoreV1().ConfigMaps("default").Get(cfgmap, metaV1.GetOptions{})
```

Comment:
This will only look for config maps in "default" namespace. Please change it to `v1.NamespaceAll`